app/src/main/java/com/example/parkingsystem_kotlin/mvp/presenter/ParkingSizeDialogPresenter.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.parkingsystem_kotlin.mvp.presenter
+
+import com.example.parkingsystem_kotlin.fragment.SpacesParkingDialogListener
+import com.example.parkingsystem_kotlin.mvp.contract.ParkingSizeDialogContract
+
+class ParkingSizeDialogPresenter(
+    private val view: ParkingSizeDialogContract.ParkingSizeView,
+) : ParkingSizeDialogContract.ParkingSizePresenter {
+
+    override fun setParkingSize(inputListener: SpacesParkingDialogListener) {
+        val parkingSize = view.getNewParkingSize()
+        try {
+            if (parkingSize.isNotEmpty() && parkingSize.toInt() < 10000) {
+                inputListener.setParkingSize(parkingSize)
+                view.closeDialog()
+            } else {
+                view.showToastEmptyInput()
+            }
+        }catch (e: NumberFormatException){
```

Comment:
I think that the try/catch is not necessary, to avoid this you can set on the dialog XML the input type as number so the user will only be able to put numbers

Reply:
Im using this try catch because when I use very big numbers it brokes the .toInt, maybe there is a way to solve that in another way, let me know :P

Reply:
done